asf-site-src/source/documentation/in-progress/precommit/plugins/nobuild.html.md

```diff
@@ -0,0 +1,46 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Name
+
+nobuild
+
+# Category
+
+Build Tool
+
+# Description
+
+A stub build tool that does not do anything to enable just static linting.
```

Comment:
A stub build tool that does not do anything. Use this if you would like to enable static linting without getting Yetus to work with your project's build system.